README.md

```diff
@@ -203,41 +203,41 @@ We use Docker in this project to assemble a different environment from different
 
 :point_right: _If you continue docker, it goes in the direction [Microservices](https://de.wikipedia.org/wiki/Microservices), Scaling and automation. We do not care, because we want to keep our Docker setup simple and use it only for local REDAXO development._
 
-### Was wird benötigt?
+### What is needed?
 
-Du musst nur [Docker (Community Edition) für dein System](https://www.docker.com/community-edition#/download) installieren, mehr wird nicht benötigt. In der Konfiguration musst du die Ordner freigeben, in denen Docker-Projekte laufen dürfen. Hier gibst du nun erstmal nur den Ordner an, in dem dieses Git-Repo liegt. Danach begibst du dich in deiner Konsole in diesen Ordner und startest die Container:
+You only have to [Docker (Community Edition) for your System](https://www.docker.com/community-edition#/download) install, more is not needed. In the configuration, you must share the folders where Docker projects are allowed to run. Here you give now only the folder, in which this Git repo lies. Then you go into this folder in your console and start the containers:
 
     $ docker-compose up -d
 
-Das wird beim ersten Mal ein kleines Weilchen dauern, weil zuerst die _Images_ runtergeladen werden müssen, aus denen Docker dann lauffähige Container baut. In deiner Konsole wird eine Menge Text vorbeilaufen.
+This will take a little while the first time, because first the _Images_ must be downloaded, from which Docker then builds working containers. There will be a lot of text in your console.
 
-:warning: Wenn die Konsole wieder bereit ist und die Befehlszeile erscheint, musst du __noch weitere 1-2 Minuten warten__, bis REDAXO vollständig installiert ist. Den Status der REDAXO-Installation siehst du nicht in deiner Konsole, weil der Vorgang im Container stattfindet. Du kannst dir die Container-Logs anschauen mittels `docker-compose logs web` (Das `web` am Ende ist unser Webserver, `db` wäre die Datenbank). Alternativ siehst du die Logs auch im kostenlosen Docker-Tool [Kitematic](https://kitematic.com), das sehr praktisch ist, wenn du mit mehreren Docker-Projekten arbeitest.
+:warning: When the console is ready again and the command line appears, you will have to wait __another 1-2 minutes__ until REDAXO is fully installed. You will not see the status of the REDAXO installation in your console because the process is taking place in the container. You can look at the container logs using `docker-compose logs web` (The `web` at the end is our web server, `db` would be the database). Alternatively you can see the logs in the free Docker tool [Kitematic](https://kitematic.com), This is very useful when working with multiple Docker projects.
```

Comment:
:warning: When the console is ready again and the command line appears, you will have to wait __another 1-2 minutes__ until REDAXO is fully installed. You will not see the status of the REDAXO installation in your console, because the process is taking place inside of the container. You can look at the container logs using `docker-compose logs web` (The `web` at the end is our web server, `db` would be the database). Alternatively you can see the logs in the free Docker tool [Kitematic](https://kitematic.com), which is very useful when working with multiple Docker projects.